lib/definition/types/keys.rb

```diff
@@ -30,15 +30,10 @@ def include(other)
           raise ArgumentError.new("Included Definition can only be a Keys Definition") unless other.is_a?(Types::Keys)
 
           ensure_keys_do_not_interfere(other)
-          other.required_definitions.each do |key, definition|
-            required(key, definition)
-          end
-          other.optional_definitions.each do |key, definition|
-            optional(key, definition)
-          end
-          other.defaults.each do |key, default|
-            default(key, default)
-          end
+
+          self.required_definitions += other.required_definitions
+          self.optional_definitions += other.optional_definitions
```

Comment:
is << maybe faster ?

Reply:
did not try it but it would require looping over it in ruby. i assume this one might be faster as it could do the looping in c but don't know how its implemented